src/panels.cpp

```diff
@@ -1440,6 +1435,127 @@ std::pair<std::string, nc_color> display::rad_badge_text_color( const Character
     return std::make_pair( rad_text, rad_color );
 }
 
+std::string display::bodypart_status( const Character &u, const bodypart_id &bp )
+{
+    if( bp == bodypart_str_id::NULL_ID() ) {
+        return "null?";
```

Comment:
Should raise exception or debug message

Reply:
Went with plain empty string instead; I don't know if this is ever likely to happen, but I don't really care if it does. Null bodyparts have no status.